src/concrete/ml/sklearn/base.py

```diff
@@ -96,7 +102,10 @@
 # Enable rounding feature for all tree-based models by default
 # Note: This setting is fixed and cannot be altered by users
 # However, for internal testing purposes, we retain the capability to disable this feature
-os.environ["TREES_USE_ROUNDING"] = "1"
+os.environ["TREES_USE_ROUNDING"] = os.environ.get("TREES_USE_ROUNDING", "1")
+
+# By default, the decision of the tree ensembles is made in clear
+os.environ["TREES_USE_FHE_SUM"] = os.environ.get("TREES_USE_FHE_SUM", "0")
```

Comment:
I thought we tried to avoid env variables? 🤔

Reply:
It's our third env variables =)

Reply:
We don't want these options to be used easily by the users, but in other hands,  we want to test them internally. 

- For `TREES_USE_ROUNDING` we want to spot any changes during concrete python updates
- For `TREES_USE_FHE_SUM` it's disabled now, but it will be enabled in the future to enhance our model's security once all the necessary tools are in place.